### PR TITLE
Create ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3758](https://cumulocity.atlassian.net/browse/DM-3758)
Original PR: [6699](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6699)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Repository device tabs updated to only show unfinished operations in their views. [GRAFT][release/cd] (#6699)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Repository device tabs only show unfinished operations
```

File: content/change-logs/device-management/ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3758
+version: 1020.6.1
+---
+Repository device tabs updated to only show unfinished operations in their views. [GRAFT][release/cd] (#6699)
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, the repository device tabs showed all operations, including finished ones, which could make it difficult for users to identify ongoing operations that required attention. With this change, the repository device tabs have been updated to only display unfinished operations in their views. This improvement streamlines the user interface and allows users to quickly focus on the operations that are currently in progress, enhancing overall usability and efficiency when managing devices in the repository.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md:17]:
```suggestion
Previously, the repository device tabs showed already completed operations that may differ from the currently installed item. With this change, the repository device tabs have been updated to only display active operations in their views.```

**Review by dafkk — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3758
+version: 1020.6.1
+---
+Previously, the repository device tabs showed already completed operations that may differ from the currently installed item. With this change, the repository device tabs have been updated to only display active operations in their views.```
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md:17]:
```suggestion
Previously, the repository device tabs showed already completed operations that may differ from the currently installed item. With this change, the repository device tabs have been updated to only display active operations in their views.
```

Comment (BeateRixen) [reply content/change-logs/device-management/ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md:17]: @dafkk What is meant with repository device tabs? Where? In the device details?

Comment (dafkk) [reply content/change-logs/device-management/ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md:17]: Repository device tabs are tab in the device details yes for all repository tabs - software device tab, firmware device tab, configuration device tab and device profile tab.

File: content/change-logs/device-management/ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3758
+version: 1020.6.1
+---
+Previously, the repository device tabs showed already completed operations that may differ from the currently installed item. With this change, the repository device tabs have been updated to only display active operations in their views.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-6-1-repository-device-tabs-updated-to-only-show-unfinished-operations-in-their.md:17]:
```suggestion
Previously, the repository device tabs in the device details showed already completed operations which could differ from the currently installed items. With this change, the repository device tabs have been updated to only display active operations in their views.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**